src/base/bittorrent/torrenthandle.cpp

```diff
@@ -1358,6 +1358,10 @@ void TorrentHandle::handleStorageMovedAlert(libtorrent::storage_moved_alert *p)
     }
 
     qDebug("Torrent is successfully moved from %s to %s", qPrintable(m_oldPath), qPrintable(m_newPath));
+    if (m_oldPath == m_session->torrentTempPath(hash())) {
+        qDebug() << "Removing torrent temp folder:" << m_oldPath;
+        Utils::Fs::removeDirRecursive(m_oldPath);
```

Comment:
I assume that we delete the temporary folder, which is owned by qBittorrent application, not the user (logically, of course).